docs/docs/features/dongle.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: Keyboard Dongle
+sidebar_label: Keyboard Dongle
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+:::warning
+This implementation is experimental and subject to change.
+:::
+
+A bluetooth dongle can be added to any keyboard running ZMK. The result is a [split keyboard](./split-keyboards.md) with the dongle as "central". There are a number of advantages to adding a dongle, but also some disadvantages:
+
+Benefits:
+
+- The "central" dongle results in increased battery life for the keyboard/former "central", as it is now a peripheral.
+- It is easier to connect to the host device.
+
+Disadvantages:
+
+- An extra microcontroller is needed.
+- The keyboard becomes unusable without the dongle.
+
+
+
+## Editing a Keyboard Definition to Support a Dongle
+
```

Comment:
Please link to https://zmk.dev/docs/faq#why-boards-and-shields-why-not-just-keyboard somewhere, stating that this guide is for keyboards which are board+shield only. Either that or you can figure out how to make it good for keyboards which are only boards as well.